recipes/weave/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "weave" %}
+{% set version = "0.15.0" %}
+{% set sha256 = "4b7ac15abe2d1c0901c33db79d4bdcad8f2261198da36a48707b0023bca72f83" %}
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py3k]  # Python2-only package
+  script: python setup.py install --record record.txt
+
+requirements:
+  build:
+    - python <3
+    - numpy
```

Comment:
Looks like `weave` is [happy working with setuptools](https://github.com/scipy/weave/blob/v0.15.0/setup.py#L190). Can we be opinionated and include `setuptools` as a build time requirement? If so, also add the `--single-version-externally-managed` flag to the `build/script:` line? [brief summary of why we prefer this setup.py flag](https://github.com/conda-forge/staged-recipes#8-when-or-why-do-i-need-to-use-python-setuppy-install---single-version-externally-managed---record-recordtxt)